observation_portal/task_scheduler.py

```diff
@@ -19,5 +21,8 @@ def run():
         expire_access_tokens.send,
         CronTrigger.from_crontab('0 15 * * *')
     )
+    scheduler.add_job(
+        time_allocation_reminder.send,
+        CronTrigger.from_crontab('0 * 1 * *')  # monthly
```

Comment:
I believe this schedule will send out an email once an hour on the first day of a month.

Reply:
Good catch. Fixed.